src/main/kotlin/no/nav/familie/klage/behandling/domain/Behandling.kt

```diff
@@ -15,28 +17,13 @@ data class Behandling(
     val status: BehandlingStatus = BehandlingStatus.OPPRETTET,
     @Embedded(onEmpty = Embedded.OnEmpty.USE_EMPTY)
     val sporbar: Sporbar = Sporbar(),
-    val resultat: BehandlingResultat? = BehandlingResultat.IKKE_SATT,
+    val resultat: BehandlingResultat = BehandlingResultat.IKKE_SATT,
     val vedtakDato: LocalDateTime? = null,
     val eksternBehandlingId: String,
     val klageMottatt: LocalDate,
     val behandlendeEnhet: String
 )
 
-enum class BehandlingResultat(val displayName: String) {
-    MEDHOLD(displayName = "Medhold"),
-    IKKE_MEDHOLD(displayName = "Ikke medhold"),
-    IKKE_SATT(displayName = "Ikke satt"),
-}
-
-enum class BehandlingStatus {
-    OPPRETTET,
-    UTREDES,
-    VENTER,
-    FERDIGSTILT;
-
-    fun erLåstForVidereBehandling(): Boolean = listOf(VENTER, FERDIGSTILT).contains(this)
```

Comment:
Denne går i beina  på OC og Mattis sin sjekk ?

Reply:
Vet ikke? Denne er flyttet til kontrakter

Reply:
Vi kan godt jobbe rundt dette - legge til unntak for systembruker på familie-kabal

Reply:
Men - metodenavnet erLåstForVidereBehandling blir kanskje litt feil nå. Den er fortsatt bare "kanskje" låst for videre behandling.

Reply:
Ønsker egentlig å flytte denne tilbake til familie-klage??

Reply:
Vi har flyttet den tilbake